### PR TITLE
More 1.26 e2e tests

### PR DESCRIPTION
**Pull request by taneyland**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/docker_test.go [hunk 1]
```diff
@@ -17,14 +17,14 @@ import (
 )
 
 // Labels
-func TestDockerKubernetes125Labels(t *testing.T) {
+func TestDockerKubernetesLabels(t *testing.T) {
```
Comment (mitalipaygude) [inline test/e2e/docker_test.go:20]: Are we not renaming this to 126Labels because there is the only test for Docker labels?

Comment (taneyland) [reply test/e2e/docker_test.go:20]: Yeah exactly & we always just update it to the latest version

**Comment by mitalipaygude** [comment]
Shouldnt we be removing all the feature flags right before the release? Is this going out in v0.15.0 release?

**Comment by taneyland** [comment]
> Shouldn't we be removing all the feature flags right before the release? Is this going out in v0.15.0 release?

Yes, I added a line to apply it right before we run the create/upgrade commands so we no longer have to specify it in every single new test

**Pull request merged**